src/kDataFrames/kDataFrameMQF.cpp

```diff
@@ -288,7 +288,7 @@ kDataFrame(frame->ksize()){
     this->class_name = "MQF"; // Temporary until resolving #17
     this->falsePositiveRate = 0.0;
     lastKmerOrder=1;
-    KD = new Kmers(kSize, frame->KD->hash_mode);
+    KD = new Kmers(kSize, integer_hasher);
```

Comment:
Why fixing the hashing mode? the kmerDecoder object is expected to be protein-hasher or strand-specific hashing.